tests/e2e/testfiles/scenarios.yaml

```diff
@@ -299,7 +299,7 @@
   paths:
     - "testfiles/storage_csm_powerscale_replica.yaml"
   tags:
-    # - "powerscale"
+    - "powerscale"
```

Comment:
Some tags got uncommented in this file, should they stay commented?

Reply:
Its good, we got it tested.